modules/user-access/pages/query-privilege-migration.adoc

```diff
@@ -0,0 +1,136 @@
+= Query Privilege Migration
+
+This page explains the procedures for the query privilege migration with an eligible version to upgrade from to 4.1.0.
```

Comment:
Not very clear. "from to"?

This page provides detailed procedures for migrating query privileges when upgrading to version 4.1.0. It also outlines the specific privileges required to successfully carry out the migration process.